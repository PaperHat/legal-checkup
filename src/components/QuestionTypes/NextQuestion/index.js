// import React from 'react'
// import styled from 'styled-components'

// const StyledNextQuestion = styled.span`
//     color:red;
//     border-style: solid;
// `

// export default function NextQuestion ({
//     handleClick
// })
