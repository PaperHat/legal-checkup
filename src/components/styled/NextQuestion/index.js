import styled from 'styled-components'

const NextQuestion = styled.span`
  background-color: #ccc;
  color: ${props => (props.next ? 'green' : 'yellow')};
  cursor: ${props => (props.next ? 'pointer' : 'not-allowed')};
  display: inline-block;
  padding: 5px;
  margin: 1px;
  border-radius: 2px;

  &:before {
    content: '>';
  }
`
NextQuestion.displayName = 'StyledNextQuestion'

export default NextQuestion
