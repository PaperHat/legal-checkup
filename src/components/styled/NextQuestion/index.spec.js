import React from 'react'
import { shallow } from 'enzyme'

import NextQuestion from '.'

describe('components:styled:NextQuestion', () => {
  it('renders the NextQuestion CSS properties properly', () => {
    expect(
      toJson(shallow(<NextQuestion>NextQuestion</NextQuestion>))
    ).toMatchSnapshot()
  })

  it('renders the NextQuestion properly with next prop', () => {
    expect(toJson(shallow(<NextQuestion next />))).toMatchSnapshot()
  })
})
