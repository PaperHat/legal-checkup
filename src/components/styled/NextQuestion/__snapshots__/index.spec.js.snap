// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components:styled:NextQuestion renders the NextQuestion CSS properties properly 1`] = `
.c0 {
  background-color: #ccc;
  color: yellow;
  cursor: not-allowed;
  display: inline-block;
  padding: 5px;
  margin: 1px;
  border-radius: 2px;
}

.c0:before {
  content: '>';
}

<span
  className="c0"
>
  NextQuestion
</span>
`;

exports[`components:styled:NextQuestion renders the NextQuestion properly with next prop 1`] = `
.c0 {
  background-color: #ccc;
  color: green;
  cursor: pointer;
  display: inline-block;
  padding: 5px;
  margin: 1px;
  border-radius: 2px;
}

.c0:before {
  content: '>';
}

<span
  className="c0"
/>
`;
